maths/monte_carlo_integration_univariate.py

```diff
@@ -0,0 +1,111 @@
+"""
+This program finds the approximate integration value/area under the curve of
+a specified function within specified limits using Monte Carlo integration method.
+
+Further, a graph of the individual areas under the curve considered for the calculation
+is also plotted. (PLOT SECTION -> Optional implementation)
+"""
+
+import doctest
+
+# importing the modules
+import random
+
+# import matplotlib.pyplot as plt
```

Comment:
```suggestion
```

Reply:
The import is necessary for the plotting section, which has been kept as an optional feature for the user.